modules/op-resolver-bundle-config.adoc

```diff
@@ -0,0 +1,35 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-bundles-config_{context}"]
+= Configuring the bundles resolver
+
+Optionally, you can change the default service account name and the default kind for pulling resources from a Tekton bundle.
```

Comment:
```suggestion
You can change the default service account name and the default kind for pulling resources from a Tekton bundle by configuring the bundles resolver.
```

I think "optionally" is redundant here, since the rest of the sentence reads "you can" rather than "you must".